doc/source/installation/windows_wsl.rst

```diff
@@ -258,18 +308,81 @@ If you have already installed deal.II and lethe, you can update them without doi
 .. code-block:: text
   :class: copy-button
 
-  cd Softwares/candi
-  ./candi.sh -j$numprocs
+	cd Software/candi
+	./candi.sh -j$numprocs
 
 2. |linux_shell| Then, update lethe:
 
 .. code-block:: text
   :class: copy-button
 
-  cd ../lethe/git
-  git pull
-  cd ../build
-  cmake ../git -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=../inst/
-  make -j$numprocs
+	cd ../lethe/git
+	git pull
+	cd ../build
+	cmake ../git -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=../inst/
+	make -j$numprocs
 	
 
+Troubleshooting
+-------------------------------------
+
+.. _change compiler default version:
+
+Change compiler default version
++++++++++++++++++++++++++++++++++++++
+
+|linux_shell| After you installed ``gcc-10``, ``g++-10`` and ``gfortran-10``, manually update default versions in the terminal:
+
+.. code-block:: text
+
+	sudo update-alternatives --remove-all gcc
+	sudo update-alternatives --remove-all g++
+	sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 10
+	sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-10 10
+	sudo update-alternatives --set cc /usr/bin/gcc
+	sudo update-alternatives --set c++ /usr/bin/g++
+
+Then, check again the version used:
+
+.. code-block:: text
+
+	gcc --version
+
+Should return ``gcc (Ubuntu 10.X.X...) 10.X.X``.
+
+
+.. _modify candi installation parameters with nano:
+
+Modify candi installation parameters with nano
++++++++++++++++++++++++++++++++++++++++++++++++
+
+|linux_shell| If you do not see the Ubuntu mount in the Windows file manager, you can modify the candi parameter files in the Ubuntu terminal directly. The procedure is:
+
+1. Open the desired file in the terminal with ``nano`` (built-in text editor):
+
+.. code-block:: text
+
+	cd <folder_name>
+	nano <file_name>
+
+.. admonition:: Example for the candi.cfg
+
+	.. code-block:: text
+
+		cd /home/<user_name>/Software/candi
+		nano candi.cfg
+
+2. Modify the text in the file, using only the keyboard. 
+
+.. tip::
```

Comment:
I'd change it for a note and put it right after line 359

Reply:
@jeannejoachim I don't have a strong opinion on this so I'll let you judge. Just tell me when you it's ready for final merge.

Reply:
Yes that's a good idea, thank you Victor :)